_pages/en_US/get-started.txt

```diff
@@ -23,42 +23,110 @@ If you see an unusual menu, STOP - you already have custom firmware! Continue fr
-    </tr>
-    <tr>
-      <td style="text-align: center; font-weight: bold;">1.0.0 to 11.3.0</td>
-      <td style="text-align: center; font-weight: bold;"><a href="installing-boot9strap-(soundhax)">Installing boot9strap (Soundhax)</a></td>
-    </tr>
-  </tbody>
-</table>
+To find the correct method for your device, please enter the system version you found in Section II.
+<div class="selectversion">
+<br>
+<select id="mySelecti">
+    <option>Ver.</option>
+    <option>other</option>
+</select>
+<select id="mySelect">
+  <option>0</option>
+  <option>1</option>
+  <option>2</option>
+  <option>3</option>
+  <option>4</option>
+  <option>5</option>
+  <option>6</option>
+  <option>7</option>
+  <option>8</option>
+  <option>9</option>
+  <option>10</option>
+  <option>11</option>
+</select>.<select id="mySelec">
+    <option>0</option>
+    <option>1</option>
+    <option>2</option>
+    <option>3</option>
+    <option>4</option>
+    <option>5</option>
+    <option>6</option>
+    <option>7</option>
+    <option>8</option>
+    <option>9</option>
+    <option>10</option>
+    <option>11</option>
+    <option>12</option>
+    <option>13</option>
+    <option>14</option>
+    <option>15</option>
+    <option>16</option>
+</select>.<select id="mySele">
+  <option>0</option>
+</select>-<select id="mySel">
+    <option>0</option>
+    <option>1</option>
+    <option>2</option>
+    <option>3</option>
+    <option>4</option>
+    <option>5</option>
+    <option>6</option>
+    <option>7</option>
+    <option>8</option>
+    <option>9</option>
+    <option>10</option>
+    <option>11</option>
+    <option>12</option>
+    <option>13</option>
+    <option>14</option>
+    <option>15</option>
+    <option>16</option>
+    <option>17</option>
+    <option>18</option>
+    <option>19</option>
+    <option>20</option>
+    <option>21</option>
+    <option>22</option>
+    <option>23</option>
+    <option>24</option>
+    <option>25</option>
+    <option>26</option>
+    <option>27</option>
+    <option>28</option>
+    <option>29</option>
+    <option>30</option>
+    <option>31</option>
+    <option>32</option>
+    <option>33</option>
+    <option>34</option>
+    <option>35</option>
+    <option>36</option>
+    <option>37</option>
+    <option>38</option>
+    <option>39</option>
+    <option>40</option>
+    <option>41</option>
+    <option>42</option>
+    <option>43</option>
+    <option>44</option>
+    <option>45</option>
+    <option>46</option>
+    <option>47</option>
+    <option>48</option>
+    <option>49</option>
+</select><select id="mySe">
+  <option>E</option>
+  <option>U</option>
+  <option>J</option>
+  <option>K</option>
+  <option>T</option>
+  <option>C</option>
+</select>
+<br>
+<input type="button" onclick="redirect()" value="Let's Go!">
+</div>
+<p id="result"></p>
+<noscript>Please enable JavaScript to continue with the guide.</noscript>
```

Comment:
Put current table here I guess maybe

Reply:
why though? the guide cant be followed without javascript

Reply:
fine whatever